components/dashboard/src/index.tsx

```diff
@@ -4,6 +4,7 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
+import "setimmediate";
```

Comment:
Is this a polyfill for the `setImmediate` global? Can we add a comment on this line explaining why it's necessary, since it's not apparent in this file as the import goes unused?

Reply:
yep, required by vscode-jsonrpc, will add a comment